Tools/SandboxTest.ps1

```diff
@@ -206,7 +212,13 @@ Write-Host @'
 --> Refreshing environment variables
 '@
 Update-EnvironmentVariables
+`$currentARP = Get-ARPTable
 
+Write-Host @'
+
+--> Comparing ARP Entries
+'@
+(Compare-Object `$currentARP `$originalARP -IncludeEqual -ExcludeDifferent).InputObject 

```

Comment:
```suggestion
(Compare-Object `$currentARP `$originalARP).InputObject 
```

This way it only shows the new ones, the way it was in your last commit it would only show the ARP entries that were on the machine before you ran `winget install`.

Example:
```
PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> (Compare-Object $currentARP $originalARP -IncludeEqual -ExcludeDifferent).InputObject

DisplayName    DisplayVersion Publisher             ProductCode
-----------    -------------- ---------             -----------
Microsoft Edge 90.0.818.66    Microsoft Corporation Microsoft Edge


PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs> (Compare-Object $currentARP $originalARP).InputObject

DisplayName                  DisplayVersion Publisher        ProductCode
-----------                  -------------- ---------        -----------
JetBrains ETW Service        212.16.26.0    JetBrains s.r.o  {DEA96022-4B1D-4D5F-B590-4625EAFFF810}
JetBrains dotTrace 2021.2.2  2021.2.2       JetBrains s.r.o. {2d5368a1-3297-5a09-87f5-79a5b4864a3c}
JetBrains dotMemory 2021.2.2 2021.2.2       JetBrains s.r.o. {41109cb0-ef67-5b7b-96d3-23797d822c32}
JetBrains dotPeek 2021.2.2   2021.2.2       JetBrains s.r.o. {546640b4-6e04-545d-94f9-f7ce6cff2993}
JetBrains dotCover 2021.2.2  2021.2.2       JetBrains s.r.o. {e0b1c9b0-9540-5e20-8a2f-3524e9a06db0}
JetBrains Rider 2021.2.2     2021.2.2       JetBrains s.r.o. {e2d2761c-f4ec-5bb3-aad5-521b644d1fdc}


PS C:\Users\WDAGUtilityAccount\Desktop\winget-pkgs>
```

Reply:
I tried that, but it doesn't work for me on PS5. It only fetches one object

Reply:
I'm... on PowerShell 5 too? Hmm. What package are you testing with?

Reply:
> I'm... on PowerShell 5 too? Hmm. What package are you testing with?

I tested with GOG.Galaxy, Google.Chrome, and Git.Git

Reply:
I think I got it this time

Reply:
![image](https://user-images.githubusercontent.com/12611259/140680709-97bfa695-443a-4beb-9ce4-da193e9996bd.png)